syntax = "proto3";

package RTB5;

message Campaign {
  //
  uint64 campaign_id = 1;
  //
  uint64 union_id = 2;
  //
  string campaign_name = 3;
  //
  string pattern = 4;
  //
  double percent = 5;
}
